src/master/matoclserv.cc

```diff
@@ -700,19 +701,23 @@ int matoclserv_load_sessions() {
 		fclose(fd);
 		return -1;
 	}
-	if (memcmp(hdr,SFSSIGNATURE "S 1.5",8)==0) {
+	if (memcmp(hdr,SFSSIGNATURE "S 1.5",8)==0 ||
+	    memcmp(hdr, (legacySessionSignature + "S 1.5").c_str(), 8) == 0) {
```

Comment:
Please, consider to use the same style as SFSSIGNATURE, I mean a macro MFSSIGNATURE, to use the same style until both are modernized.

Reply:
I remember our last discussion about code style was, because we never find time to apply a full code refactor we would try to introduce new code using latest style even if it means having mixed styles in some part of the code. This is why i avoid to use c-style macros and use std::string type. I also wanted to scope the changes only to the part of the code that is using it rather that expose it to global `SFSCommunication.h` because this feature is intended to be temporal and will probably be deprecated after we drop support for migration at some point.

Reply:
I agree that this is temporal. But, please notice that you are still doing the same, using memcmp and converting the string to an array to compare it. The lines are just longer.

Reply:
applied